notebooks/mock-notebooks/login/User Registration.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
<p>Suggestion:</p><ul><li>A user with this email already exists on { node.name }. You can try logging in by...(login instructions)</li></ul><p> </p><p>Reason:</p><ul><li>May be nice to remind the user which domain they are trying to register to</li><li>Would be nice to remind them on how to login since they seem to already have an account</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/6333/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>